.github/workflows/prerequisite-test.yml

```diff
@@ -0,0 +1,14 @@
+name: Verify there is matching build of security plugin
```

Comment:
This name appears in the checks like this:

![Screen Shot 2022-08-24 at 4 44 09 PM](https://user-images.githubusercontent.com/17432265/186519818-0812bfad-2dd5-407a-b2ed-fef9d9687d4b.png)

Shorten this to "Prerequisite Checks"?